js/blog-tags.js

```diff
@@ -1,4 +1,5 @@
 import { buildTaxonomiesUrl, fetchKontent, baseUrl } from "./kontent-fetching.js";
+import { getUrlParamater } from "./app.js";
```

Comment:
This the best place for this function?

Reply:
Probably not, but at the moment we don't have any other generic file, such as utils.js